gwlfe/enums.py

```diff
@@ -18,14 +18,14 @@ def parse(value):
 
 
 class ETflag(object):
-    HAMMON_METHOD = '<Hammon method>'
+    HAMON_METHOD = '<Hammon method>'
```

Comment:
The GMS file actually uses `0` to indicate the Hamon method. These string values exist purely for symbolic reasons. But you're right, this should be "Hamon".